automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala

```diff
@@ -91,15 +90,11 @@ trait NotebookTestUtils extends LeonardoTestUtils {
     }
   }
 
-  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
   def withNewNotebook[T](cluster: Cluster, kernel: NotebookKernel = Python3, timeout: FiniteDuration = 2.minutes)(testCode: NotebookPage => T)(implicit webDriver: WebDriver, token: AuthToken): T = {
     withNotebooksListPage(cluster) { notebooksListPage =>
-      val result: Future[T] = retryUntilSuccessOrTimeout(whenKernelNotReady, failureLogMessage = s"Cannot make new notebook")(30 seconds, 2 minutes) {() =>
-        Future(notebooksListPage.withNewNotebook(kernel, timeout) { notebookPage =>
-          testCode(notebookPage)
-        })
+      notebooksListPage.withNewNotebook(kernel, timeout) { notebookPage =>
```

Comment:
I haven't proved this but I suspect this Future-based retry wasn't playing nice with selenium WebDriver. Tests were failing around this `withNewNotebook` call. Removing the Future & retry to see if the situation improves.

Reply:
Were you ever able to reproduce the issues Gary saw?

Reply:
I wasn’t - I ran tests many times and they all passed for me. :(